jobs/kpi-forecasting/kpi_forecasting/tests/test_prophet_forecast.py

```diff
@@ -1,11 +1,544 @@
 import pandas as pd
 from dotmap import DotMap
 import numpy as np
+import pytest
+import collections
 
 
 from kpi_forecasting.models.prophet_forecast import ProphetForecast
 
 
+@pytest.fixture
+def forecast():
+    A1_start_date = "2124-01-01"
```

Comment:
The dates "2124-01-01" and "2124-01-02" have a ["magic number"](https://refactoring.guru/replace-magic-number-with-symbolic-constant) feel -- they gets used so frequently they should probably be defined as a global parameter at the top, like:
```
FIRST_FUTURE_DATE = "2024-01-01"
SECOND_FUTURE_DATE = "2024-01-02"
```
and then referenced throughout the tests. Maybe different variable names than I've used though... naming things is hard.

Reply:
That's a good idea.  My concern is that if there are too many dates it will impact readability but I don't think that happens enough it is a major concern?  I'll take a look and see what I can do, the "magic number feel" is a good point